packages/app/components/jsonschemaform/widgets/AnchorWidget.tsx

```diff
@@ -0,0 +1,44 @@
+import type { WidgetProps } from '@rjsf/utils'
+import { useEffect, useState } from 'react'
+import { findAndReplace, macros } from './utils'
+
+/**
+ * Schema Layout:
+    "MyProperty": {
+        "ui:widget": "anchor",
+        "ui:options": {
+            "href": "http://localhost:8080/service-request-from-config?variable_entry_id=******",
+            "text": "",
+            "change": {
+                "every": "***",
+                "to": ["FIELD_LOOKUP:EntryID", "FIELD_LOOKUP:VariableEntryID"] // replace with no regex, not replaceAll, is used. This means you can stack "every" to replace per "to" field.
+            } 
+        }
+    },
+ */
+export default function AnchorWidget({ options }: WidgetProps) {
+    const [href, setHref] = useState(options.href as string)
+
+    useEffect(() => {
+        // @ts-expect-error
+        if (!!options.change.every) {
```

Comment:
If the instance of this widget (declared in your model's schema) has the `change.every` field, run a find and replace with macro support.